packages/browser-agent-core/src/features/spa/aggregate/index.js

```diff
@@ -183,11 +184,16 @@ export class Aggregate extends FeatureBase {
 
       if (!state.pageLoaded && evName === 'load' && eventSource === window) {
         state.pageLoaded = true
+        state.pageLoadTime = now()
```

Comment:
Nice work on this! Is this import used anywhere?

Reply:
What import?

Reply:
nvm I got you....one sec